iidm/iidm-api/src/main/java/com/powsybl/iidm/network/LoadingLimits.java

```diff
@@ -44,6 +44,8 @@ interface TemporaryLimit {
          * @return false if it is a real limit, false otherwise
          */
         boolean isFictitious();
+
+        void setValue(double temporaryLimitValue);
```

Comment:
I don't see a problem about that. For me the use case is the one in the tests: get the current limits, change one temporary limit value because the data is invalid. So I think `LoadingLimits.setTemporaryLimitValue` is enough.

Reply:
Ok, thanks!
I think this method should be removed then.